File: app/src/main/java/br/com/diogomoreno/events/presentation/board/BoardActivity.java
package br.com.diogomoreno.events.presentation.board;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import br.com.flasco.leanworks.R;
import br.com.diogomoreno.events.presentation.board.ui.BoardFragment;
import dagger.hilt.android.AndroidEntryPoint;

@AndroidEntryPoint
public class BoardActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.board_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, BoardFragment.newInstance())
                    .commitNow();
        }
    }
}

File: app/src/main/java/br/com/diogomoreno/events/presentation/board/BoardActivity.kt
package br.com.diogomoreno.events.presentation.board

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import br.com.diogomoreno.events.presentation.board.ui.BoardFragment
import br.com.flasco.leanworks.R
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class BoardActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.board_activity)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.container, BoardFragment.Companion.newInstance())
                    .commitNow()
        }
    }
}